public/index.html

```diff
@@ -20,6 +20,11 @@
       href="https://cdnjs.cloudflare.com/ajax/libs/material-design-iconic-font/2.2.0/css/material-design-iconic-font.min.css"
     />
 
+    <link
+      rel="stylesheet"
+      href=""
```

Comment:
why did you need to add this tag referencing to nothing?

Reply:
resolved